app/services/analytics.rb

```diff
@@ -51,7 +49,7 @@ def uuid
   INVALID_AUTHENTICITY_TOKEN = 'Invalid Authenticity Token'.freeze
   INVALID_SERVICE_PROVIDER = 'Invalid Service Provider'.freeze
   OTP_DELIVERY_SELECTION = 'OTP: Delivery Selection'.freeze
-  OTP_RESULT = 'OTP: Result'.freeze
+  MFA = :mfa
```

Comment:
Yep. I was thinking we would use all symbols eventually, but now I think strings are better when selecting them from a dropdown on a site like keen.io or Mixpanel.